modules/mixcr/1.0/mixcr.smk

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Anita dos Santos
+# Module Author:    Anita dos Santos
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["mixcr"]`
+CFG = op.setup_module(
+    name = "mixcr",
+    version = "1.0",
+    subdirectories = ["inputs", "mixcr", "outputs"],
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _install_mixcr,
+    _mixcr_input_fastq,
+    _mixcr_output_txt,
+    _mixcr_all,
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _mixcr_input_fastq:
+    input:
+        fastq_1 = CFG["inputs"]["sample_fastq_1"],
+        fastq_2 = CFG["inputs"]["sample_fastq_2"],
+    output:
+        fastq_1 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R1.fastq.gz",
+        fastq_2 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R2.fastq.gz",
+    run:
+        op.relative_symlink(input.fastq_1, output.fastq_1)
+        op.relative_symlink(input.fastq_2, output.fastq_2)
+
+# Installs latest MiXCR release from github if the mixcr folder is not present yet
+rule _install_mixcr:
+    params:
+        mixcr = CFG["lcr-modules"] + CFG["conda_envs"]["mixcr"]
+    output: 
+        complete = CFG["lcr-modules"] + CFG["conda_envs"]["mixcr"] + "/mixcr_dependencies_installed.success"
+    shell:
+        '''
+        latest_tag=$(curl --silent "https://api.github.com/repos/milaboratory/mixcr/releases/latest" |  grep '"tag_name":'| sed -E "s/v//" | sed -E 's/.*\"([^\"]+)\".*/\\1/');
```

Comment:
This seems convoluted. Can't you just use the latest as-is without knowing the tag name?

Reply:
You're right, I've made some changes